docs/headless-chrome.md

```diff
@@ -1,28 +1,62 @@
-# Running headless Chrome for Lighthouse
+# Running Lighthouse using headless Chrome
 
-The headless_shell still has a few bugs to work out. Until then, Chrome + xvfb is a stable solution.
```

Comment:
are we confident enough in headless parity already to remove the xvfb documentation entirely?